packages/ui-signer/src/Extrinsic.tsx

```diff
@@ -3,30 +3,47 @@
 // of the ISC license. See the LICENSE file for details.
 
 import { I18nProps } from '@polkadot/ui-app/types';
+import { RxFees } from '@polkadot/api-observable/types';
 import { QueueTx } from './types';
+// TODO - reuse instead of obtaining from app-transfer
```

Comment:
100% on this TOD. Should rather have it in signer (and let Transfer pull that from that package), instead of having the dependency from ui into app.